kokoro/gcp_ubuntu/bazel/bindings/build_kokoro.sh

```diff
@@ -27,13 +27,13 @@ export PS4='[$(date -u "+%T %Z")] '
 WORKDIR="${KOKORO_ARTIFACTS_DIR?}/github/iree"
 
 # Mount the checked out repository, make that the working directory and run the
-# tests in the bazel-tensorflow image.
-# TODO: Change image to gcr.io/iree-oss/bazel-bindings:prod once we create it.
+# tests in the bazel-bindings image.
+# TODO(meadowlark): Replace :latest with :prod before submitting!
```

Comment:
I recommend a "DO NOT SUBMIT" comment for code you need to change before submitting. There are internal checks that will looks for it

Reply:
Good to know!